packages/fx-core/src/plugins/resource/identity/utils/telemetryUtil.ts

```diff
@@ -8,34 +8,52 @@ export class TelemetryUtils {
         TelemetryUtils.ctx = ctx;
     }
 
+    private static send(eventName: string,
+        properties: { [key: string]: string; },
+        measurements?: { [key: string]: number; }) {
+        properties[Telemetry.properties.component] = Telemetry.componentName;
+        if (this.ctx.app.id) {
+            properties[Telemetry.properties.appid] = this.ctx.app.id;
+        }
+        TelemetryUtils.ctx.telemetryReporter?.sendTelemetryEvent(eventName, properties, measurements);
+    }
+
     public static sendEvent(eventName: string,
         properties?: { [key: string]: string; },
         measurements?: { [key: string]: number; }) {
         if (!properties) {
             properties = {};
         }
-        properties[Telemetry.component] = Telemetry.telemetryName;
-        TelemetryUtils.ctx.telemetryReporter?.sendTelemetryEvent(eventName, properties, measurements);
+        TelemetryUtils.send(eventName, properties, measurements);
     }
 
-    public static sendErrorEvent(eventName: string,
+    public static sendSuccessEvent(eventName: string,
```

Comment:
this method seems could be merged with sendEvent?

Reply:
It is fixed. Thx.